build/yamls/flow-aggregator/flow-aggregator.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
```

Comment:
This manifest needs to be integrated with `hack/generate-manifest.sh`.

In particular `generate-manifest.sh` should support generating a release version of the manifest for which the docker image tag is the correct release tag. This manifest should be automatically uploaded to release assets when a Github release is created (so you need to edit `upload_release_assets.yml`).

Make sure that you double check everything and that you test `hack/generate-manifest.sh` locally, otherwise it breaks the release flow.